chpl/chpl-service/src/main/java/gov/healthit/chpl/dto/DeveloperStatusEventDTO.java

```diff
@@ -130,6 +131,19 @@ public boolean equals(final Object obj) {
         return Objects.equals(getId(), dto.getId());
     }
 
+    @Override
+    public int hashCode() {
+        final int prime = 31;
```

Comment:
The "equals" method in this class is only checking the ID field. It's my understanding that equals and hashcode have to always return consistently (i.e. if equals returns true for two objects then they had better have the same hashcode). While I'm sure it's very unlikely that they would be different, shouldn't either equals be changed to check all these fields or hashcode be changed to only check the ID?